### PR TITLE
Updating Brand Assets

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:** This PR adds the Logos and its ideologies to the document.
**Screenshot:** 

Related to Issue #33

### CODE REVIEW

File: docs/branding/patterns-illustrations.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
```
Comment (palisadoes) [inline docs/branding/patterns-illustrations.md:1]: Delete this file as there is no meaningful content.

File: docs/branding/color-palette.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (palisadoes) [inline docs/branding/color-palette.md:1]: Delete this file as there is no meaningful content

File: docs/branding/logos.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (palisadoes) [inline docs/branding/logos.md:1]: Add a reference to the color palette and illustrative PNG to this file. You have an appropriate palette section at the bottom where this should be placed.

File: docs/branding/typography.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (palisadoes) [inline docs/branding/typography.md:1]: Delete this file as there is no meaningful content. Add a typography section to the logos.md file. So many different files are not necessary.

File: sidebars.js [hunk 5]
```diff
@@ -68,5 +68,6 @@ module.exports = {
       ]
     }],
     GSoC: [ 'gsoc/gsoc-ideas' ],
+    "Brand Assets": ["branding/logos", "branding/color-palette", "branding/typography", "branding/patterns-illustrations"],
```
Comment (palisadoes) [inline sidebars.js:71]: Change the heading from "Brand Assets" to Branding. Move the section above GSoC

File: docs/branding/logos.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+## Idea
+Our logo is made up of 2 shapes, circle & rectangle, it represents,
+
+1. "**Talawa**" in the Jamaican dialect means "**Strong**", and there is a saying "**Little, but Talawa**", which translates to "**Little, but Strong**".
+2. We used Palisadoes Foundation logo's elements and colors to preserve the core ideologies.
+
+## Clear space
+The logo needs to have clear space, free of text, pictures, and other elements.
+
+![Clear Space](./assets/Logo-Guidelines.PNG)
```
Comment (palisadoes) [inline docs/branding/logos.md:21]: There is no reference to any of the existing logos. A number were created and submitted in a previous PR, which appears to have been canceled. Please add.

**Review by palisadoes — COMMENTED** [review]
A good start. There a number of changes that need to be made.

**Comment by adithyaakrishna** [comment]
> A good start. There a number of changes that need to be made.

@palisadoes Thank You, I will work on the changes asap and commit them soon :)

File: docs/branding/patterns-illustrations.md (hunk 1 above)
Comment (adithyaakrishna) [reply docs/branding/patterns-illustrations.md:1]: I'm working on the files without the content, I will complete all of them and push a commit to this PR once they are done 💯

File: docs/branding/logos.md (hunk 3 above)
Comment (adithyaakrishna) [reply docs/branding/logos.md:1]: Okay got it, will resolve this immediately

File: sidebars.js (hunk 5 above)
Comment (adithyaakrishna) [reply sidebars.js:71]: Okay sure 💯

File: docs/branding/logos.md (hunk 3 above)
Comment (palisadoes) [reply docs/branding/logos.md:1]: Rename this file branding.md. The logos should go on a separate page.

**Comment by palisadoes** [comment]
There are a number of branch conflicts. Please resolve.

**Pull request closed without merging**